README.md

```diff
@@ -1,78 +1,96 @@
-# Android Video Ahoy App
+# Twilio Video Android App
+
+[![CircleCI](https://circleci.com/gh/twilio/twilio-video-app-android.svg?style=shield&circle-token=40105862db5fbadac9bdcff973375d5114417b66)](https://circleci.com/gh/twilio/twilio-video-app-android)
 
 This application demonstrates multi-party voice and video built with [Twilio’s Programmable Video Android SDK](https://www.twilio.com/docs/video).
 
-- [Languages](#languages)
-- [Getting Started](#getting-started)
-    - [Build](#build)
-    - [Unit Tests](#unit-tests)
-- [Contributing](#contributing)
-- [Code of Conduct](#code-of-conduct)
-- [License](#license)
+![video-app-screenshots](https://user-images.githubusercontent.com/1930363/76543029-867ec080-644b-11ea-8145-d15d3fe9f7ea.png)
+
+## Features
+
+- [x] Video conferencing with real-time video and audio
+- [x] Enable/disable camera
+- [x] Mute/unmute mic
+- [x] Switch between front and back camera
+- [x] [Dominant speaker](https://www.twilio.com/docs/video/detecting-dominant-speaker) indicator
+- [x] [Network quality](https://www.twilio.com/docs/video/using-network-quality-api) indicator
+
+## Languages
```

Comment:
@Alton09 this is the only top level section that is not consistent with iOS and it is a really small section so I wonder if it could be inside another section?

Reply:
Yeah good point. We can move it after the test section.